packages/flame/lib/effects.dart

```diff
@@ -30,6 +30,7 @@ export 'src/effects/provider_interfaces.dart'
     show
         AnchorProvider,
         AngleProvider,
+        ReadOnlyAngleProvider,
```

Comment:
Can you export the `ReadOnlySizeProvider` too? 😇

Reply:
Symmetry… so satisfactory.